homeassistant/components/homeassistant_hardware/strings.json

```diff
@@ -18,18 +18,29 @@
             "uninstall_addon": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::step::uninstall_addon::title%]"
           }
         },
+        "change_channel": {
+          "title": "[%key:component::homeassistant_hardware::silabs_multiprotocol_hardware::options::step::reconfigure_addon::title%]",
+          "data": {
+            "channel": "Channel"
+          }
+        },
         "install_addon": {
           "title": "The Silicon Labs Multiprotocol add-on installation has started"
         },
         "notify_channel_change": {
           "title": "Channel change initiated",
           "description": "A Zigbee and Thread channel change has been initiated and will finish in {delay_minutes} minutes."
         },
+        "notify_no_multipan_user": {
+          "title": "Manual configuration may be needed",
+          "description": "Home Assistant found nothing using the IEEE 802.15.4 radio. If you have configured a service to use the radio yourself, for example Zigbee2MQTT, you will have to reconfigure the channel in that service after completing this guide."
```

Comment:
Maybe clarify what we were expecting to find but got "nothing"? Is it a channel, a network, a device, or something else?

Reply:
Right, we should say something like "Home Assistant found no integrations.."
We actually specifically know about zha and otbr, so we could inject/name those maybe.